ci-operator/step-registry/baremetal/lab/etcd-encryption/OWNERS

```diff
@@ -0,0 +1,13 @@
+approvers:
```

Comment:
This can be a symlink of file `../OWNERS`

Reply:
it tested etcd encryption step in this PR, but intend to use on next PR for:
UPI etcd encryption, disk mirroring and disk encryption